.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended'],
+  parser: '@typescript-eslint/parser',
+  plugins: ['@typescript-eslint', 'cypress'],
+  root: true,
+  rules: {
+    '@typescript-eslint/no-var-requires': 0,
```

Comment:
Is this rule still necessary?

Reply:
Still useful to disable rules for test files. Let me tweak the rules.

Reply:
Since the test code isn't typescript, maybe its better to have a separate eslint config in the tests directory. That way you're not applying typescript rules to non-typescript files.

Reply:
Done